web/js/layers/wv.layers.modal.js

```diff
@@ -32,85 +32,105 @@ wv.layers.modal = wv.layers.modal || function(models, ui, config) {
   var $selectedCategory = $(self.selector + " #selected-category");
   var $allLayers = $(self.selector + " #layers-all");
   var gridItemWidth = 320; //with of grid item + spacing
+  var projection = models.proj.selected.id;
   var modalHeight;
   var sizeMultiplier;
   var searchBool;
+  var hasSetting;
 
-  // Visible Layers
+  //Visible Layers
```

Comment:
It's a good idea to avoid whitespace changes when possible, since they make it harder to figure out when the last meaningful change was made using `git blame`.

Reply:
Definitely. Hm, this must have been caused by an autofixer somewhere; probably after the switch from 4 to 2 spaces. I prefer the space after the `//` so I wouldn't have changed this willingly. I'm going to do another round or two of code cleanup so I'll go through and revert these before my next commit.